test/parallel/test-stream-preprocess.js

```diff
@@ -0,0 +1,71 @@
+﻿/******************************************************************************/
+'use strict';
+/******************************************************************************/
+const common = require('../common');
+const assert = require('assert');
+
+const fs = require('fs');
+const path = require('path');
+const rl = require('readline');
+/******************************************************************************/
```

Comment:
Should it be this way?
```js
'use strict';

const common = require('../common');
const assert = require('assert');

const fs = require('fs');
const path = require('path');
const rl = require('readline');

const BOM = '\uFEFF';
```
Should I also replace other asterisk lines with  blank lines?